src/model/treewalker.js

```diff
@@ -90,6 +90,8 @@ export default class TreeWalker {
 			this.position = Position.createFromPosition( this.boundaries[ this.direction == 'backward' ? 'end' : 'start' ] );
 		}
 
+		this.position.stickiness = 'toNone';
```

Comment:
Comment needed that it is to reset the stickiness.

Reply:
👍